### ISSUE TITLE
iOS doesn't close the popup window after login

### ISSUE THREAD

**Issue by elhamsarikhani** [MrSwitch/hello.js#339]
Hi,
I'm using ionic view app for testing my app and everything works fine in Android but in iOS on both safari as well as chrome, it freezes after successfully logging in and doesn't close the pop-up window.

**Comment by MrSwitch** [comment]
Have you solved this already?

**Comment by elhamsarikhani** [comment]
Yes the issue was with ionic view testing env. as I build the app and ran it on device the issue was solved on its own. :) I reported this to ionic forum and they said it's due to the fact that inAppBrowser plugin is not supported in Ionic View.
